tools/recog_test_imgs.sh

```diff
@@ -0,0 +1,26 @@
+#!/usr/bin/env bash
```

Comment:
I don't understand why this bash wrapper is needed.
Python scripts have better help tools. You can type `./tools/recog_test_imgs.py -h` to see which argument is needed.

Reply:
unnecessary